tests/Aspire.Confluent.Kafka.Tests/ProducerConfigurationTests.cs

```diff
@@ -0,0 +1,151 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Text;
+using Confluent.Kafka;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Xunit;
+
+namespace Aspire.Confluent.Kafka.Tests;
+
+public class ProducerConfigurationTests
+{
+    [Theory]
+    [InlineData(true)]
+    [InlineData(false)]
+    public void ReadsFromConnectionStringsCorrectly(bool useKeyed)
+    {
+        var builder = Host.CreateEmptyApplicationBuilder(null);
+        builder.Configuration.AddInMemoryCollection([
+            new KeyValuePair<string, string?>("ConnectionStrings:messaging", CommonHelpers.TestingEndpoint)
+        ]);
+
+        if (useKeyed)
+        {
+            builder.AddKeyedKafkaProducer<string, string>("messaging");
+        }
+        else
+        {
+            builder.AddKafkaProducer<string, string>("messaging");
+        }
+
+        var host = builder.Build();
+        var connectionFactory = useKeyed ?
+            host.Services.GetRequiredKeyedService(ReflectionHelpers.ProducerConnectionFactoryStringKeyStringValueType.Value!, "messaging") :
+            host.Services.GetRequiredService(ReflectionHelpers.ProducerConnectionFactoryStringKeyStringValueType.Value!);
+
+        ProducerConfig config = GetProducerConfig(connectionFactory)!;
+
+        Assert.Equal(CommonHelpers.TestingEndpoint, config.BootstrapServers);
+    }
+
+    [ConditionalTheory]
```

Comment:
```suggestion
    [Theory]
```

For all the tests in this file.

Reply:
done